specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1451 @@
+        }
+      },
+      "put": {
+        "tags": [
+          "NetworkVerifier"
+        ],
+        "summary": "Creates/Updates the specific Network Verifier resource",
+        "operationId": "NetworkVerifierPut",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The Network Verifier resource object to create/update",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "NetworkVerifier was updated",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          },
+          "201": {
+            "description": "NetworkVerifier was created",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkVerifierPut": {
+            "$ref": "./examples/NetworkVerifierPut.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "NetworkVerifier"
+        ],
+        "summary": "Updates the specific Network Verifier resource",
+        "operationId": "NetworkVerifierPatch",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The Network Verifier resource object to create/update",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
```

Comment:
you cannot patch the full object , it would be a subset  of properties that must include "tags" .. For example top level arm properties like id, name, type etc cannot be patched

Reply:
Updated PATCH method to only include properties that can be updated.